src/map/script.cpp

```diff
@@ -8269,6 +8269,10 @@ BUILDIN_FUNC(getpartymember)
 				ShowError("buildin_getpartymember: The array %s is not string type.\n", varname);
 				return SCRIPT_CMD_FAILURE;
 			}
+			if (not_server_variable(*varname)) {
+				ShowError("buildin_getpartymember: Requires a server variable, player variable '%s' is invalid.\n", varname);
```

Comment:
For me blocking player variable is overkill. How about `script_rid2sd` check like https://github.com/rathena/rathena/blob/c5a5c545181bcb1dae9b33131acf137f5033a59f/src/map/script.cpp#L6035